Rick-and-Morty/Rick And Morty/Factories/CharacterCardStateFactory.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CharacterCardStateFactory.swift
+//  Rick And Morty
+//
+//  Created by Scottie Gray on 2021-08-13.
+//  Copyright © 2021 Novoda. All rights reserved.
+//
+
+import Foundation
+
+final class CharacterCardStateFactory {
+    private func getIsAlive(character: Character) -> Bool {
+        if character.status == "Alive" {
+            return true
+        }
+        
+        return false
+    }
+    
+    private func getFirstEpisode(character: Character) -> String? {
+        if let firstEpisode = character.episodeURLs.first {
+            return firstEpisode
+        }
+        
+        return nil
+    }
```

Comment:
If you're returning nil this code doesn't do anything. you could just call `character.episodeURLs.first`